web/public/fonts.css

```
@@ -0,0 +1,558 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)